apps/modernization-ui/src/apps/page-builder/page/management/header/tabs/PageTabs.tsx

```diff
@@ -29,6 +95,94 @@ export const PageTabs = ({ tabs }: Props) => {
                     {tab.name}
                 </li>
             ))}
+            {onAddSuccess ? (
+                <>
+                    <ModalToggleButton unstyled type="button" modalRef={modalRef} data-testid="openManageTabs">
+                        <Icon.Edit />
+                        <h4>Manage tabs</h4>
```

Comment:
Issue: we should avoid using `h` tags unless we are specifying a header over a body of text. a `Heading` component exists that accepts a `level` prop that will provide the styling.